### PR TITLE
Nc news 4

### PR DESCRIPTION
**Pull request by YusufKader**

### CODE REVIEW

**Review by yalicee — CHANGES_REQUESTED** [review]
Missing 404 not found error handling test

File: app.js [hunk 1]
```diff
@@ -10,14 +10,20 @@ app.get("/api/articles/:article_id", getArticlesById);
...
+});
 
-  return res.status(500).send({ message: "Internal Server Error" });
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ message: "Invalid data type" });
+  } else next(err);
+});
+app.use((err, req, res, next) => {
+  console.log(err);
```
Comment (leekli) [inline app.js]: Remove console log

File: models/articles.model.js [hunk 2]
```diff
@@ -4,9 +4,17 @@ exports.selectArticlesById = (article_id) => {
...
+
+      // if (data.rowCount === 0) {
+      //   return Promise.reject({
+      //     status: 404,
+      //     message: "article not found",
+      //   });
+      // }
+      // if (typeof article_id !== "string") {
+      //   return Promise.reject({ status: 400, message: "Invalid input" });
+      // }
```
Comment (leekli) [inline models/articles.model.js]: Remove all this commented out code if you aren't going to use it

**Review by leekli — APPROVED** [review]
Approved, feel free to merge.

Please do remove the console logs and commented out code though before you merge anything.

**Pull request merged**